manifests/09_operator-ibm-cloud-managed.yaml

```diff
@@ -26,7 +26,8 @@ spec:
         seccompProfile:
           type: RuntimeDefault
       serviceAccountName: marketplace-operator
-      nodeSelector: {}
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
```

Comment:
Isn't this supposed to now be `.../control-plane`? See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#urgent-upgrade-notes

I expect that OCP will respect both labels for probably eternity, but we should at least introduce the new label as well.